packages/shopify-codemod/transforms/remove-redundant-returns-from-addeventlistener.js

```diff
@@ -0,0 +1,28 @@
+export default function removeAddEventListener(file, api) {
+  const j = api.jscodeshift;
+  return j(file.source)
+    .find(j.CallExpression, {
+      callee: {
+        type: 'MemberExpression',
+        property: {
+          name: 'addEventListener',
+        },
+      },
+    })
+    .forEach((nodes) => {
```

Comment:
`nodes` is a misleading name because an array isn't being passed in here, and it's actually a `NodePath`.

I'd recommend using destructuring to keep it concise. e.g., `.forEach(({node}) => {`